client/my-sites/plans-features-main/media-queries.tsx

```diff
@@ -0,0 +1,31 @@
+import { css } from '@emotion/react';
+import type { SerializedStyles } from '@emotion/react';
+
+const sidebarWidth = 272; //in px
+const plansPageSmallBreakpoint = '780px';
+const plansPageSmallWithSidebarBreakpoint = `${ 780 + sidebarWidth }px`;
```

Comment:
Same here with this `780` value. Do we have a pattern for defining these dimensions in the owner component?

Reply:
Good point. I think this is a custom breakpoint we've been using, but a globally defined one might exist (it might exist as a CSS constant only though, somewhere).

Merits some investigation. Potentially some common layout component ought to export these.